README.md

```diff
@@ -17,29 +17,29 @@ Additional access is granted via [snap interfaces](https://snapcraft.io/docs/int
 
 After the installation it's necessary to connect the interfaces:
 
+- [cpu-control](https://snapcraft.io/docs/cpu-control-interface)
+- `etc-default-grub` plug into the [system-files](https://snapcraft.io/docs/system-files-interface) interface;
 - [hardware-observe](https://snapcraft.io/docs/hardware-observe-interface)
 - [home](https://snapcraft.io/docs/home-interface)
```

Comment:
Given the removal of `plugs.home.read: all`, the home interface will auto connect on classic.

```suggestion
- [home](https://snapcraft.io/docs/home-interface) - auto connected on classic distributions
```